constants/ourTeam.tsx

```diff
@@ -0,0 +1,91 @@
+import Atharv from '@assets/atharv.png'
+import Aditya from '@assets/aditya.png'
+import Ekta from '@assets/ekta.png'
+import Geo from '@assets/geoffrey.png'
+import Pranshu from '@assets/pranshu.jpeg'
+import Karthik from '@assets/karthik.png'
+import Kushagra from '@assets/kushagra.png'
+import Shrestha from '@assets/shrestha.png'
+import Shubham from '@assets/shubham.png'
+import Sparsh from '@assets/sparsh.png'
+import Tushar from '@assets/tushar.png'
```

Comment:
Please refrain from adding these images directly to the assets folder; instead, move them to the `assets/team` directory.

Reply:
ok